### PR TITLE
Try bluer bg colors for pivot tables

### PR DESCRIPTION
**Pull request by mazameli**
This makes pivot tables look a bit more like our design mockups.

## Before

image

## After

image

### CODE REVIEW

**Comment by kdoh** [comment]
Did you test this out w/ dark mode on dashboards?

**Comment by mazameli** [comment]
Hadn't, but it looks like this:

image

Detail:

image

File: frontend/src/metabase/visualizations/visualizations/PivotTable.css [hunk 1]
```diff
@@ -0,0 +1,6 @@
+:local .PivotBGLight {
```
Comment (kdoh) [inline frontend/src/metabase/visualizations/visualizations/PivotTable.css:1]: If possible I'd like to not keep creating new css modules code so we can eventually rip it out and get rid of yet another CSS solution. I'd just define two new variables in the Pivot Table file and use our `metabase/lib/color` library there to get this effect.

**Review by kdoh — CHANGES_REQUESTED** [review]
I'm not sure I'm 100% feeling that blue tint in dark mode but It's relatively minor all things considered. Do think we should simplify the implementation of this though and just do this inside of the pivot table file itself since these are truly one off CSS classes we wouldn't use anywhere else.

**Comment by mazameli** [comment]
@kdoh, I've killed the separate CSS module and got the in-file CSS styles working.

**Review by kdoh — APPROVED** [review]

**Pull request merged**